.github/workflows/OS-UnitTests.yml

```diff
@@ -13,6 +13,10 @@ jobs:
       fail-fast: true
       matrix:
         os: [ubuntu-latest, windows-latest, macos-latest]
+        version:
+          - '1.4'
+          - '1.5'
+          - '1.6.0'
```

Comment:
There's a 1.6.1 now. IIRC, simply making this '1.6' should include this version too...